packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -112,23 +110,19 @@ export class SpinButton extends React.Component<ISpinButtonProps, ISpinButtonSta
   public componentWillUnmount(): void {
     this._async.dispose();
   }
-
   /**
    * Invoked when a component is receiving new props. This method is not called for the initial render.
    */
   // tslint:disable-next-line function-name
   public UNSAFE_componentWillReceiveProps(newProps: ISpinButtonProps): void {
     this._lastValidValue = this.state.value;
-    let value: string = newProps.value !== undefined ? newProps.value : String(newProps.min);
-    if (newProps.defaultValue) {
-      value = String(Math.max(newProps.min as number, Math.min(newProps.max as number, Number(newProps.defaultValue))));
-    }
 
     if (newProps.value !== undefined) {
       this.setState({
-        value: value
+        value: newProps.value
```

Comment:
This behavior makes sense (not coercing to min/max), but it might trip someone up who is fresh to the code. Would be nice to have a test that explicitly highlights the fact that this is an allowed case.